src/backend/utils.py

```diff
@@ -51,6 +51,8 @@ def get_dot_distance(vector1, vector2):
 def get_cosine_similarity(vector1, vector2):
     return spatial.distance.cosine(vector1, vector2)
 
+def get_euclidian_distance(vector1, vector2):
```

Comment:
this function is already present above at line 30

Reply:
dafaq mere waale me it was missing.... had to write it again..... looks like ur changes came